src/compas_timber/design/wall_from_surface.py

```diff
@@ -285,17 +287,20 @@ def parse_loops(self):
                     polyline_points.append(edge.start_vertex.point)
                 else:
                     polyline_points.append(edge.end_vertex.point)
+                length += edge.length
             polyline_points.append(polyline_points[0])
+            offset_dist = length * 0.001
```

Comment:
ideally use the tolerance module. but if it doesn't quite give you what you want the at least name these numbers